blockchain/beacon/src/drand.rs

```diff
@@ -53,6 +54,29 @@ pub struct DrandConfig<'a> {
     pub chain_info: ChainInfo<'a>,
     /// Network type
     pub network_type: DrandNetwork,
+    // Tokio Runtime config
+    pub tokio: TokioConfig,
+}
+
+#[derive(serde::Deserialize, serde::Serialize, PartialEq, Eq, Clone)]
+pub struct TokioConfig {
+    pub worker_threads_number: usize,
+    pub max_number_of_blocking_threads: usize,
+    pub blocking_thread_keep_alive_timeout: Duration,
+    pub thread_stack_size: usize,
+    pub global_queue_interval: u32,
+}
+
+impl Default for TokioConfig {
+    fn default() -> Self {
+        Self {
+            worker_threads_number: 4,
```

Comment:
It would be better to use same values than for the default constructor.

Reply:
Fixed